README.md

```diff
@@ -399,8 +399,8 @@ The following table describes the set of configuration options for the Okta prov
 | ----- | ---------- | -------- | ----- |
 | `credentialsSecret` | Reference to a secret containing authentication details (See below) | `''`  | Yes |
 | `groups` | List of groups to filter against | `nil`  | No |
-| `url` | Okta organization URL (Must contain a trailing slash) | `''`  | Yes |
-| `appId` | Application ID of App Groups are assigned to | `''`  | Yes |
+| `url` | Okta URL which can be found under the "Okta Domain" in your application settings (must contain the protocol and a trailing slash) | `''`  | Yes |
```

Comment:
Optional, but it looks like for URLs this component is called "scheme" instead: https://en.wikipedia.org/wiki/URL#Syntax
```suggestion
| `url` | Okta URL which can be found under the "Okta Domain" in your application settings (must contain the scheme and a trailing slash) | `''`  | Yes |
```